packages/auth/graphql/resolvers/User.js

```diff
@@ -1,8 +1,9 @@
 const Roles = require('../../lib/Roles')
+const userAccessRoles = ['admin', 'supporter', 'editor']
```

Comment:
IMO editor should be a general user access role—just email

Reply:
agree and changed